_posts/en/newsletters/2018-07-31-newsletter.md

```diff
@@ -0,0 +1,177 @@
+---
+title: 'Bitcoin Optech Newsletter #6'
+permalink: /en/newsletters/2018/07/31/
+name: 2018-07-31-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter includes the usual dashboard and action items, a
+feature article by developer Anthony Towns about the consolidation of
+four million UTXOs at Xapo, news about possible upgrades to Bitcoin's
+script system, links to a few highly-voted questions and answers on
+the Bitcoin StackExchange, and some notable commits in the development
+branches of the Bitcoin Core, Lightning Network Daemon (LND), and
+C-lightning projects.
+
+## Action items
+
+{% comment %}<!-- TODO: replace with something else if 0.16.2 not released -->{% endcomment %}
+
+- **Bitcoin Core 0.16.2 released:** a minor release that brings bug
+  fixes and minor improvements.  If you use the [abandontransaction][rpc
+  abandontransaction] or [verifytxoutproof][rpc verifytxoutproof] RPCs,
+  you should particularly consider upgrading.  Otherwise, we recommend
+  you review the [release notes][bitcoin core 0.16.2] for other changes
+  that may affect your operation and upgrade when convenient.
+
+## Dashboard items
+
+- **Fees still low:** hash rate increased difficulty by more than 10%
+  in the 2,016-block retarget period ending Sunday, giving an average
+  time between blocks of less than 9 minutes and helping to absorb the
+  increased transaction load from the past week's price speculation.
+  However, after the transition away from the normal weekend transaction
+  lull into the new week with average time between blocks restored to 10
+  minutes, there's the possibility for a rapid increase estimated
+  transaction fees.  We recommend being careful sending large low-fee
+  transactions such as consolidations until closer to the weekend when
+  transaction volume begins to taper off again.
```

Comment:
This is really good. I think the phrasing can be tweaked slightly to make it read better. I'll try to do that in a fixup commit.